File: library/src/main/java/com/luck/picture/lib/animators/ViewHelper.java
package com.luck.picture.lib.animators;

import android.view.View;

import androidx.core.view.ViewCompat;

/**
 * @author：luck
 * @date：2020-04-18 14:13
 * @describe：ViewHelper
 */
public final class ViewHelper {
    public static void clear(View v) {
        v.setAlpha(1);
        v.setScaleY(1);
        v.setScaleX(1);
        v.setTranslationY(0);
        v.setTranslationX(0);
        v.setRotation(0);
        v.setRotationY(0);
        v.setRotationX(0);
        v.setPivotY(v.getMeasuredHeight() / 2);
        v.setPivotX(v.getMeasuredWidth() / 2);
        ViewCompat.animate(v).setInterpolator(null).setStartDelay(0);
    }
}

File: selector/src/main/java/com/luck/picture/lib/animators/ViewHelper.kt
package com.luck.picture.lib.animators

import android.view.View
import androidx.core.view.ViewCompat

/**
 * @author：luck
 * @date：2020-04-18 14:13
 * @describe：ViewHelper
 */
object ViewHelper {
    fun clear(v: View) {
        v.alpha = 1f
        v.scaleY = 1f
        v.scaleX = 1f
        v.translationY = 0f
        v.translationX = 0f
        v.rotation = 0f
        v.rotationY = 0f
        v.rotationX = 0f
        v.pivotY = (v.measuredHeight shr 1).toFloat()
        v.pivotX = (v.measuredWidth shr 1).toFloat()
        ViewCompat.animate(v).setInterpolator(null).startDelay = 0
    }
}